src/h5web/visualizations/heatmap/ColorMapSelector.tsx

```diff
@@ -22,38 +19,34 @@ const DropdownIndicator = (props: any): JSX.Element => {
 // So I use the color map name as both label and value
 const colorMapOptions = Object.keys(INTERPOLATORS).map(label => ({ label }));
 
-function ColorMapSelector(props: Props): JSX.Element {
-  const { className } = props;
-
-  const { colorMap } = useHeatmapState();
-  const { setColorMap } = useHeatmapActions();
+function ColorMapSelector(): JSX.Element {
+  const colorMap = useHeatmapStore(state => state.colorMap);
+  const setColorMap = useHeatmapStore(state => state.setColorMap);
```

Comment:
You can write a selector that returns more than one thing: https://github.com/react-spring/zustand#selecting-multiple-state-slices

... on second thought, the syntax is pretty heavy, so maybe not. :(